storage/client/client.go

```diff
@@ -1621,7 +1634,7 @@ func (c *StorageClient) RuntimeTokens(ctx context.Context, p apiTypes.GetRuntime
 	return &ts, nil
 }
 
-func (c *StorageClient) RuntimeTokenHolders(ctx context.Context, p apiTypes.GetRuntimeEvmTokensAddressHoldersParams, address staking.Address) (*TokenHolderList, error) {
+func (c *StorageClient) RuntimeTokenHolders(ctx context.Context, p apiTypes.GetRuntimeEvmTokensAddressHoldersParams, address *staking.Address) (*TokenHolderList, error) {
```

Comment:
changed for consistency with the rest of the api; if this was intentionally not a pointer please lmk

Reply:
wait everything else uses pointer?

Reply:
oh yeah most runtime ones use address

Reply:
oh the address param is required for this endpoint; which is why some endpoints use pointers. reverting this.